lib/bank_account/savings_account.rb

```diff
@@ -0,0 +1,28 @@
+module Bank
+  class SavingsAccount < Account
+
+    def initialize(id, initial_balance)
+      super
+      raise ArgumentError.new "Insufficient Funds" unless initial_balance >= 10
+    end
+
+    # charges $2 for every withdrawal
+    def withdraw(amount)
+      @balance -= (amount + 2)
+      if @balance < 10
+        puts "Savings can not fall below $10.Your remaining savings amount is: "
+        return @balance += (amount + 2)
+      else
+        return @balance
+      end
+    end
```

Comment:
This works, but just as a FYI (and I'm not necessarily advocating this), this method could have been written like this to avoid duplicating very similar code in your `Account` class:
```ruby
def withdraw(amount)
  amount += 2
  super
end
```